### ISSUE TITLE
UX Is the App already tracking?

### ISSUE THREAD

**Issue by mrascher-cosee** [corona-warn-app/cwa-documentation#250]
## Describe the issue
I opened the App an read all information. After that i came to the main screen. And then there was no information about my tracking. Is the tracking already on? Should i activate something? 

## Suggested change
Probably you need a bigger indicator for an active tracking. Like a radar or some animation. After that the users knows the tracking is active and it will find all persons around me.

**Comment by WilliamReed2004** [comment]
Yes it's already tracking, it just doesn't have enough information because only approx. 5000 People in Germany have this app downloaded until now, so the chance of being near someone with the app is quite low.

**Comment by niklas2810** [comment]
You can find out if the tracing is active in the background by checking the app. At the top of the screen, "Exposure Logging Active" should be displayed. Otherwise, an error symbol (e.g. "Bluetooth turned off") will be shown. 

As far as I am concerned, the notifications are handled by the API provided by Google/Apple, therefore the CWA has no influence on the notifications and only display the current state inside the app. However, you can test the notifications by disabling bluetooth or your location services, in this case a notification should pop up as well :wink:

**Comment by WilliamReed2004** [comment]
If Bluetooth is turend off while tracking is enabled a notification similar to this one will appear
![Screenshot_20200616-090958](https://user-images.githubusercontent.com/29880819/84743175-dc668580-afb1-11ea-86e0-951dffbad514.png)

**Comment by mrascher-cosee** [comment]
yeah you are all correct, but remember there are people out there, which are not so confirmed with all the techniques. You get information about if somthing is turned off thats true but there is no information about "hey everthing is fine, everything is correct installed, we track in the background"...

**Comment by Faldrian** [comment]
I have a similar error message. I have bluetooth active and the app is showing that the tracking is enabled on the top of the app.

But android 10 or the framework on the phone is warning me that tracking is not happending? Why does it seem like I need location data to be enabled?
![Screenshot_20200616-092804](https://user-images.githubusercontent.com/1184859/84745064-513abf00-afb4-11ea-8059-b9de967f219a.png)

**Comment by Leseratte10** [comment]
As far as I know, Google grouped "Bluetooth Low Energy", "GPS location" and "cellular location" into one group "Location data". 
The app doesn't use GPS or cell locations (as seen by the app permissions and/or the source code), but the generic location setting on the phone needs to be enabled.

**Comment by Faldrian** [comment]
So can someone confirm that I can't use the app / the exposure tracking if I don't want location services to be on (because I do not want my device to know where I am)? I'm more than happy to have bluetooth enabled, because it is on anyways (smartwatch...).

**Comment by niklas2810** [comment]
Yeah, unfortunately you will have to enable location services. I guess this will lead to a lot of confusion in the future ("they are tracking our locations!!!!"), but that's nothing the devs can change...

**Comment by Faldrian** [comment]
So my suggestion would be: Clearly communicate why this is necessary etc. on the corona page.
Further: The App is still showing me "everything ok", even if I disable location. This is a serious bug.

**Comment by lennarto** [comment]
I also don't get a notification if bluetooth is disabled (iOS). The app is still showing "everything ok".

**Comment by tkowark** [comment]
@lennarto Did you disable the tracking by turning off bluetooth in the settings or via the homescreen? For me, disabling bluetooth via settings is shown in the app, "disconnecting bluetooth devices until tomorrow" (what you're actually doing on the homescreens) seems to not trigger that. Either way, would be great if you can create an issue in the iOS repo for that.

**Comment by lennarto** [comment]
@tkowark Ah that's correct, disabling bluetooth in the settings triggers the warning, however, disabling bluetooth in the control center won't. I created an [issue](https://github.com/corona-warn-app/cwa-app-ios/issues/635)

**Comment by tekook** [comment]
@lennarto that is by design!
Toggling off Bluetooth in control center does not actually disable the Bluetooth chip. Only connection like media and stuff. 

LE connection like smart watches and the tracing are still enables. Thus the app is correct to not display a warning
